### DISCUSSION TITLE
Sass:math issue

### DISCUSSION THREAD

**Discussion by MeherabTheLord** [community/community#44502] [Other Feature Feedback, Questions, & Ideas]
Hi,
While i was trying to @use "sass:math"; module in scss file this was not working. This shows some error.I belive i donot have the access for build in module. I attached some screenshots. At that moment i need some help for solving this dilemma.Also that syntax was not highlighting.
firstImage:
![sassMathissue](https://user-images.githubusercontent.com/118060921/212740616-0637c1a6-213b-4141-bace-17a2f42b0862.JPG)
secondImage:
![sassMathissue2-ConvertImage](https://user-images.githubusercontent.com/118060921/212740666-4c02246a-1a2f-44dd-a75e-0b6d20f565d8.jpg)

**Comment by MeherabTheLord** [comment]
My problem is solved. Thank you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MeherabTheLord ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. But glad to see this has resolved 🌟
